Lombiq.BaseTheme/Lombiq.BaseTheme.csproj

```diff
@@ -49,11 +38,8 @@
     <PackageReference Include="OrchardCore.ContentManagement" Version="1.8.2" />
     <PackageReference Include="OrchardCore.DisplayManagement" Version="1.8.2" />
     <PackageReference Include="OrchardCore.ResourceManagement" Version="1.8.2" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.Attributes\Lombiq.HelpfulLibraries.Attributes.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="true" />
-    <ProjectReference Include="..\..\..\Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.SourceGenerators\Lombiq.HelpfulLibraries.SourceGenerators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+    <PackageReference Include="Lombiq.HelpfulLibraries.SourceGenerators" Version="9.1.1-alpha.15.osoe-819" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+    <PackageReference Include="Lombiq.HelpfulLibraries.Attributes" Version="9.1.1-alpha.15.osoe-819" OutputItemType="Analyzer" ReferenceOutputAssembly="true" />
   </ItemGroup>
```

Comment:
Similarly to above, we need to support both local and NuGet forms.
Add the two deleted lines to [this](https://github.com/Lombiq/Orchard-Base-Theme/pull/112/files#diff-451606a82c362f6658ece354c67b290d424846c084dbc8fbe2bc0f5af37198d7R45-R49) `<ItemGroup>`.
Move the two new lines into [this](https://github.com/Lombiq/Orchard-Base-Theme/pull/112/files#diff-451606a82c362f6658ece354c67b290d424846c084dbc8fbe2bc0f5af37198d7R51-R54) `<ItemGroup>`.